components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodManager.kt

```diff
@@ -199,6 +205,105 @@ class GitpodManager : Disposable {
         }
     }
 
+    private val portsObserveJob = GlobalScope.launch {
+        if (application.isHeadlessEnvironment) {
+            return@launch
+        }
+
+        // Ignore ports that aren't actually used by the user (e.g. ports used internally by JetBrains IDEs)
+        val ignorePorts = listOf(5990, 5991, 6679, 6942, 6943)
```

Comment:
I think we should only ignore a port of JB backend itself.

You can get it like that: https://github.com/gitpod-io/gitpod/blob/fea2135ad5e85757b510665710c9a6bfdbe3c734/components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodManager.kt#L106

you can add this port + 1 as well if JB_DEV is true

We can ignore only if we know something from us is running otherwise we should not exlude a case that a user app will do it.

Do you know who is owning 5990, 5991, 6679?

Reply:
@akosyakov I've added those to the ignore list because as soon as I start `./launch-dev-server.sh` and join the session, I get a notification for each of those ports. 

![Screenshot 2022-05-26 at 08 38 19](https://user-images.githubusercontent.com/2318450/170499242-517adc53-0291-4b08-a556-c9bbf664d217.png)

6942 seems related to JetBrains too [[1]](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360003708940-PyCharm-binds-to-a-port-that-I-need-to-use-)

I think 5990, 5991 are related to CodeWithMe [[1]](https://www.jetbrains.com/help/idea/faq-about-code-with-me-security.html#communication_ports)

`6679` indeed I wasn't able to find any reference today - so I've removed it already.

I have a feeling that most users will be annoyed by these notifications if we don't ignore them. We could suggest to add `onOpen: ignore` but it feels an extra effort for the user.

Reply:
Do you get them in preview envs as well?

Reply:
> Do you get them in preview envs as well?

No. Removed as agreed. I've only left the IDE backend port 🙏